### PR TITLE
Rename tutorials files

### PR DESCRIPTION
**Pull request by christiam**
This renames the source files for the large megablast example and the
submit-and-wait-for-results script.

### CODE REVIEW

File: docs/source/tutorials/submit-and-wait-script.rst [hunk 1]
```diff
@@ -18,70 +18,46 @@
...
-Once the above command returns (which may take a few minutes), you can check the status of the search:
+You can run it with the following command:
 
 .. code-block:: bash
 
-    elastic-blast status --cfg hepatitis.ini --loglevel DEBUG
+    ./submit-and-wait-for-results.sh ${YOUR_INI_FILE} ${TIMEOUT_IN_MINUTES}
 
-Once your search is done, you may download the results as shown below.
+The second parameter (TIMEOUT_IN_MINUTES) is optional and specifies the timeout mentioned above. The default value is 500. The configuration file (YOUR_INI_FILE) is a standard ElasticBLAST configuration file.
```
Comment (boratyng) [inline docs/source/tutorials/submit-and-wait-script.rst:43]:
```suggestion
The second parameter (TIMEOUT_IN_MINUTES) is optional and specifies the timeout mentioned above. The default value is 500 minutes. The configuration file (YOUR_INI_FILE) is a standard ElasticBLAST configuration file.
```

**Review by boratyng — APPROVED** [review]

**Comment by christiam** [comment]
@boratyng - FTR I didn't modify the files at all, simply renamed them. 

I created #71 to address your feedback.

**Comment by boratyng** [comment]
👍

**Review by boratyng — APPROVED** [review]

**Review by boratyng — APPROVED** [review]

File: docs/source/tutorials/large-megablast.rst [hunk 2]
```diff
@@ -18,46 +18,70 @@
...
-As noted earlier, ElasticBLAST will provide your results in multiple gzipped files, one for each batch it processed.  Some users prefer to have all results in one file.  You can accomplish this easily with the command below, which will gunzip and concatenate all the gzipped files in the current directory starting with ``batch`` into the file MYRESULTS.tsv.  This command will leave the original gzipped files in place. 
+Finally, make sure to delete your resources:
 
 .. code-block:: bash
 
-    gunzip -c batch_*.gz > MYRESULTS.tsv
+    elastic-blast delete --cfg hepatitis.ini --loglevel DEBUG
+
+
+You should also run the checks outlined in the quickstart to double-check that all resources have been deleted.
```
[the remark this answers is not in the document]
Comment (christiam) [reply docs/source/tutorials/large-megablast.rst:87]: @boratyng - please see #101

File: docs/source/tutorials/large-megablast.rst [hunk 3]
```diff
@@ -18,46 +18,70 @@
...
-.. _tutorial_submit_and_wait_script:
+.. _tutorial_mb:
 
-submit-and-wait-for-results script
-==================================
+MegaBLAST on a large nucleotide set
+===================================
 
-.. _script: https://github.com/ncbi/elastic-blast-demos/blob/master/submit-and-wait-for-results.sh
+In this example, you search 87,374 hepatitis sequences against the nt database, producing tabular output.  The search should take about 75 minutes and cost less than $10.  The preemptible or spot price could be as little as 20% of that, but may take longer to complete.
```
[the remark this answers is not in the document]
Comment (christiam) [reply docs/source/tutorials/large-megablast.rst:26]: @boratyng - please see #101

**Pull request merged**